ethclient/gethclient/gethclient_test.go

```diff
@@ -327,3 +328,51 @@ func testCallContract(t *testing.T, client *rpc.Client) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 }
+
+func TestOverrideAccountMarshal(t *testing.T) {
+	om := map[common.Address]OverrideAccount{
+		common.Address{0x11}: OverrideAccount{},
+		common.Address{0xaa}: OverrideAccount{
+			Nonce: 5,
+		},
```

Comment:
Well, we kind of already have that test because it's the same as not specifying the nonce at all...

Reply:
Yes right now it is -- and that's the behaviour that I want to be backed by test, so it doesn't change

Reply:
Hmm, maybe we misunderstood each other. I mean, since `0` is the zero value of `uint64`, it is already set to zero in every test except this one. If we add a test with `Nonce: 0` it will be exactly the same as the `{}` empty object test.

Reply:
>  If we add a test with Nonce: 0 it will be exactly the same as the OverrideAccount{} empty object test.

Yes, right now it is. But if we change `OverrideAccount` so that the types (including `Nonce`) are nillable, then the tests won't detect that change, just keep working. Whereas if you add an explicit `Nonce: 0`, then the test will flag the behavioural change.

Reply:
I pushed it